src/main/java/com/dev/movietheatre/Main.java

```diff
@@ -1,18 +1,43 @@
 package com.dev.movietheatre;
 
 import com.dev.movietheatre.lib.Injector;
+import com.dev.movietheatre.model.CinemaHall;
 import com.dev.movietheatre.model.Movie;
+import com.dev.movietheatre.model.MovieSession;
+import com.dev.movietheatre.service.CinemaHallService;
 import com.dev.movietheatre.service.MovieService;
+import com.dev.movietheatre.service.MovieSessionService;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
 
 public class Main {
     private static Injector injector = Injector.getInstance("com.dev.movietheatre");
+    private static final LocalDate LOCAL_DATE = LocalDate.of(2021,02,01);
```

Comment:
I think we don't need to make it a constant

Reply:
done